docs/querying/datasource.md

```diff
@@ -302,12 +302,15 @@ use join datasources.
 SQL joins take the form:
 
 ```
-<o1> [ INNER | LEFT [OUTER] ] JOIN <o2> ON <condition>
+<o1> [ INNER | LEFT [OUTER] | CROSS ] JOIN <o2> ON <condition>
 ```
 
-The condition must involve only equalities, but functions are okay, and there can be multiple equalities ANDed together.
-Conditions like `t1.x = t2.x`, or `LOWER(t1.x) = t2.x`, or `t1.x = t2.x AND t1.y = t2.y` can all be handled. Conditions
-like `t1.x <> t2.x` cannot currently be handled.
+JOIN conditions have the following requirements:
+- You can only use equality statements in JOIN conditions. For example: `t1.x = t2.x`
+- You can use functions on the operands in JOIN conditions. For example: `LOWER(t1.x) = t2.x`
+- You can apply multiple equality conditions using the AND operator. For example: `t1.x = t2.x AND t1.y = t2.y`
+- Inequality conditions are unsupported. For example: `t1.x <> t2.x`
```

Comment:
This is redundant with the "you can only use equality".

Reply:
```suggestion
```